shared/data/src/main/java/com/ivy/data/model/Transaction.kt

```diff
@@ -89,10 +91,36 @@ fun Transaction.getAccountId(): UUID =
         is Transfer -> fromAccount.value
     }
 
+fun Transaction.getAccount(): AccountId = when (this) {
```

Comment:
Ideally, we'd like to get rid of all of these functions and have only the data" in the model but it's fine for now. Can you move these is some other/package file called migration utils or sth. The reason for that is that I don't want to couple data with behavior. 

Data should remain pure data and open to any interpretation (behavior). So just move these in some other file + package. Maybe create an `temp.migration` package and inside it `TempMigrationUtils.kt`